public/js/login.js

```diff
@@ -0,0 +1,59 @@
+$(document).ready(() => {
+  console.log("login page connected");
```

Comment:
for production ready code, you generally want to remove `console.log` from your code